test/unit/revision_test.rb

```diff
@@ -15,6 +15,21 @@ class RevisionsTest < ActiveSupport::TestCase
     assert_not_equal 0, node_revision.timestamp
     assert_not_nil node_revision.timestamp
   end
+  
+  test 'create a bad revision' do
+    node = Node.new(uid: users(:bob).id,
+                    type: 'page')
+    node.title = 'My new node for revision testing'
+    assert node.save!
+    # in testing, uid and id should be matched, although this is not yet true in production db
+    node_revision = Revision.new(title: 'My new node',
+                                 body: 'I hope this test works -- by which I mean fails! 😅',
```

Comment:
If this causes a test failure, let's try to fix it with https://github.com/publiclab/plots2/pull/7172

```ruby
        # prevent vid non-unique bug in https://github.com/publiclab/plots2/issues/7062
        raise ActiveRecord::Rollback if !node.valid? || node.vid == 0
```